vspace/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -1,112 +1,129 @@
-			Login:
-			<input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
-  			<input placeholder="Username" class="form-control input-sm" type="text" id="username" name="username"/>        
-		    <input placeholder="Password" class="form-control input-sm" type="password" id="password" name="password"/>    
-		    <button type="submit" class="btn btn-default btn-sm">Log in</button>
-		</form>
-		</sec:authorize>
-        </p>
-        </div>
-        </div>
-      </div>
-    </footer>
-    
-
-     </body>
+
+<script
+	src="//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="<c:url value="/resources/bootstrap/js/bootstrap.min.js" />"></script>
+<script src="<c:url value="/resources/bootstrap/js/main.js" />"></script>
+</head>
+
+<body>
+	<div class="container" style="padding-bottom: 150px;">
+
+		<div class="page-header">
+			<nav>
+				<ul class="nav nav-pills pull-right">
+					<li role="presentation"><a href="<c:url value="/" />">Home</a>
+					</li>
+
+					<sec:authorize access="isAuthenticated()">
+						<li role="presentation"><a
+							href="<c:url value="/staff/space/list" />">Spaces</a></li>
+						<li role="presentation">
+							<form action="<c:url value="/logout" />" method="POST">
+								<input type="hidden" name="${_csrf.parameterName}"
+									value="${_csrf.token}" />
+								<button class="btn-link" type="submit" title="Logout">
+									<i class="fa fa-sign-out" aria-hidden="true"></i> Logout
+								</button>
+							</form>
+						</li>
+					</sec:authorize>
+				</ul>
+
+			</nav>
+
+			<h1>
+				<a class="appName" href="<c:url value="/" />">My new webapp</a>
+			</h1>
+			<p class="lead">This is awesome.</p>
+		</div>
+
+
+		<c:if test="${show_alert}">
+			<div class="alert alert-${alert_type}" role="alert">${alert_msg}</div>
+		</c:if>
+		<tiles:insertAttribute name="content" />
+
+	</div>
+	<!-- /container -->
+
+	<footer class="footer">
+		<div class="container">
+
+			<div class="row">
+				<div class="col-md-12">
+					<hr style="margin-bottom: 25px;">
+					<p class="text-muted pull-left">
+					<p class="text-muted">
+
+
+						<sec:authorize access="isAnonymous()">
+
+							<form name='f' class="form-horizontal pull-right"
+								action="<c:url value="/login/authenticate" />" method="POST">
+								Login: <input type="hidden" name="${_csrf.parameterName}"
+									value="${_csrf.token}" /> <br> <input
+									placeholder="Username" class="form-control input-sm"
+									type="text" id="username" name="username" /> <br> <input
+									placeholder="Password" class="form-control input-sm"
+									type="password" id="password" name="password" />
+								<button type="submit" class="btn btn-default btn-sm">Log
+									in</button>
+							</form>
+						</sec:authorize>
+					</p>
+				</div>
+			</div>
+		</div>
+	</footer>
+
+
+</body>
```

Comment:
please avoid reformatting all of this. Did you actually change anything in here?